wordpress-mobile/AztecEditor-Android
# Issue/328 split multiline heading

### Fix
Applying the Paragraph format to a multiline heading removes format from the selected line and retains the format in the lines above and below as described in https://github.com/wordpress-mobile/AztecEditor-Android/issues/328.

### Test
1. Enter the following in HTML mode.
```
<h2 foo="bar">First
Second
Third
</h2>
```
2. Place cursor anywhere in "Second" text.
3. Tap ***Heading*** format button.
4. Tap ***Paragraph*** menu option.
5. Notice "First" is ***Heading 2*** as expected.
6. Notice "Second" is ***Paragraph*** as expected.
7. Notice "Third" is ***Heading 2*** as expected.
8. Tap ***HTML*** format button.
9. Notice HTML below.
```
<h2 foo="bar">First</h2>
Second
<h2 foo="bar">Third</h2>
```
